client/PRUserial485/consts.py

```diff
@@ -0,0 +1,68 @@
+# General
```

Comment:
consts must be the same for both server and client application. 
Should be the same file (or symbolic link)